### PR TITLE
Shadowlands/HallsOfAtonement/Trash: Add "Powerful Swipe"

### PR DESCRIPTION
**Pull request by okazan**
Hey dont know how to use github nor do i know lua but I frankensteined what I could with what I do know.

lmk how i did or if im wasting my time, just attempted this for fun. Trying to be the change I want to see in the world lmao

tyty

### CODE REVIEW

File: Shadowlands/HallsOfAtonement/Locales/esES.lua [hunk 1]
```diff
@@ -9,4 +9,5 @@ if L then
 	L.collector = "Recolectora depravada"
 	L.shard = "Fragmento de Halkias"
 	L.reaver = "Atracador natopiedra"
+	L.slasher = "Stoneborn Slasher"
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Locales/esES.lua:12]: Revert this change.

File: Shadowlands/HallsOfAtonement/Options/Colors.lua [hunk 2]
```diff
@@ -40,4 +40,5 @@ BigWigs:AddColors("Halls of Atonement Trash", {
 	[326607] = "red",
 	[344993] = "blue",
 	[346866] = "yellow",
+	[326997] = "yellow",
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Options/Colors.lua:43]: These files are auto-generated, revert this change.

File: Shadowlands/HallsOfAtonement/Options/Sounds.lua [hunk 3]
```diff
@@ -40,4 +40,5 @@ BigWigs:AddSounds("Halls of Atonement Trash", {
 	[326607] = {"alert","warning"},
 	[344993] = {"alert","warning"},
 	[346866] = "alarm",
+	[326997] = "alert",
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Options/Sounds.lua:43]: These files are auto-generated, revert this change.

File: Shadowlands/HallsOfAtonement/Trash.lua [hunk 4]
```diff
@@ -19,7 +19,8 @@ mod:RegisterEnableMob(
 	165414, -- Depraved Obliterator
 	165529, -- Depraved Collector
 	164557, -- Shard of Halkias
-	167612 -- Stoneborn Reaver
+	167612, -- Stoneborn Reaver
+	167607 -- Stonebron Slasher
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Trash.lua:23]: There's a typo in the mob's name.

File: Shadowlands/HallsOfAtonement/Trash.lua [hunk 5]
```diff
@@ -220,3 +225,9 @@ function mod:TurnToStone(args)
 	self:Message(args.spellId, "red", CL.casting:format(args.spellName))
 	self:PlaySound(args.spellId, self:Interrupter() and "warning" or "alert")
 end
+
+-- Stoneborn Slaher
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Trash.lua:229]: Typo.

File: Shadowlands/HallsOfAtonement/Trash.lua [hunk 6]
```diff
@@ -220,3 +225,9 @@ function mod:TurnToStone(args)
...
+function mod:PowerfulSwipe(args)
+	self:Message(args.spellId, "yellow", CL.casting:format(args.spellName))
```
Comment (Zireael-N) [inline Shadowlands/HallsOfAtonement/Trash.lua:231]: We use `purple` for tank abilities.

**Review by Zireael-N — CHANGES_REQUESTED** [review]

**Comment by okazan** [comment]
Did i do it? IDK how to operate github like AT ALL so sorry if im burning ur time lol

much appreciated

**Comment by Zireael-N** [comment]
Thanks!

**Pull request merged**